LinkedList.java

```diff
@@ -0,0 +1,94 @@
+/**
+ * Created by ioana-chirca on 16-May-17.
+ */
+public class LinkedList {
+    public class Node {
+        private int value;
+        private Node next;
+
+        public Node() {}
+
+        public Node(int value) {
+            this.value = value;
+        }
+
+        public void setValue(int value) {
+            this.value = value;
+        }
+
+        public int getValue() {
+            return value;
+        }
+
+        public void setNext(Node next) {
+            this.next = next;
+        }
+
+        public Node getNext() {
+            return next;
+        }
+    }
+
+    private Node head;
+    private int size;
+
+    public LinkedList() {
+    }
+
+    public int getSize() {
+        return size;
+    }
+
+    public void addLast(int value) {
```

Comment:
Exactly. I leave it up to you which way you prefer to go. But imagine you were building a huge test case, then the time spent building the list could actually matter a lot.